source/_components/microsoft.markdown

```diff
@@ -70,6 +70,14 @@ contour:
   type: string
 {% endconfiguration %}
 
+Languages supported by this integration.
```

Comment:
Instead of copying over the content from the source we should simply add a link to https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/microsoft/tts.py#L20 in the `description:`. This way we don't need to maintain the list here too.

Reply:
Updated as requested